.github/workflows/deploy.yml

```diff
@@ -24,7 +25,31 @@ jobs:
 
       - name: Deploy
         env:
-          LP_USER: ${{ secrets.LP_USER }}
-          LP_PASSWORD: ${{ secrets.LP_PASSWORD }}
-          LP_UA_EMAIL: ${{ secrets.LP_UA_EMAIL }}
+          WIKI_USER: ${{ secrets.LP_USER }}
+          WIKI_PASSWORD: ${{ secrets.LP_PASSWORD }}
+          WIKI_UA_EMAIL: ${{ secrets.LP_UA_EMAIL }}
+        run: bash deploy_dev.sh "${{ steps.changed-files.outputs.all_changed_files }}"
+
+  dev-deploy:
+    name: Dev Wikis
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Whilst we're at it, we could update this to `v3`. The `fetch-depth exists there too, so should be fine`

Reply:
Well, rto update dependencies, someone needs to go and do it ^^